portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/ExposeLanguageFromAssetLibraryAndSite.testcase

```diff
@@ -0,0 +1,91 @@
+@component-name = "portal-headless-frontend-infrastructure"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Headless Frontend Infrastructure";
+
+	setUp {
+		TestCase.setUpPortalInstanceNoSelenium();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONDepot.deleteDepot(depotName = "Test Depot Name");
+
+			Site.tearDownCP();
+		}
+	}
+
+	@priority = "4"
+	test CanRecieveCurrentSettingLanguageFromAssetLibrary {
+		property portal.acceptance = "true";
+
+		task ("Given an AssetLibrary with a list of languages configured") {
+			JSONDepot.addDepot(depotName = "Test Depot Name");
+
+			DepotNavigator.openDepotAdmin();
+
+			DepotNavigator.gotoDefineCustomLanguages(depotName = "Test Depot Name");
+
+			Button.click(button = "Edit");
+
+			Depot.configureCurrentLanguages(
+				currentLanguages = "English (United States),Chinese (China),French (France),Spanish (Spain)",
+				defaultDepotLanguage = "English (United States)");
+
+			Button.click(button = "Save");
+
+			DepotNavigator.openDepotAdmin();
```

Comment:
Seems this step is not required, please help check and remove it :D

Reply:
I think we need it, I tested it If we don't use it, the test failed on deleting a depot.

Reply:
As shared though slack, we need to move it to tearDown since only tearDown requires this step, please help check :heart: